src/react-query/utils/normalizeStatus.ts

```diff
@@ -6,7 +6,11 @@ export const normalizeStatus = (
     status: QueryStatus,
     fetchStatus: FetchStatus,
 ): DataLoaderStatus => {
-    if (status === 'loading' && fetchStatus === 'idle') {
+    if (status === 'pending') {
+        if (fetchStatus === 'fetching') {
+            return 'loading';
```

Comment:
Why didn't you use `pending`?

Reply:
`loading` should tell to draw a loader, it's like `isLoading` boolean flag is `useQuery` result. `pending` has a little bit different meaning